modules/serverless-quarkus-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-quarkus-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-quarkus-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your Quarkus functions. This allows you to modify the timing of health checks performed on the function.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a function by using `kn func create`.
+
+.Procedure
+
+. Add the `readinessProbe` and `livenessProbe` sections with new time values to the `container` section of the `func.yaml` file:
```

Comment:
Also maybe it could have some "see also" section that links or point developer to the quarkus docs https://quarkus.io/guides/smallrye-health about health checks.